.github/dependabot.yml

```diff
@@ -0,0 +1,24 @@
+version: 2
+registries:
+  npm-registry-registry-npmjs-org:
+    type: npm-registry
+    url: https://registry.npmjs.org
+    token: "${{secrets.NPM_REGISTRY_REGISTRY_NPMJS_ORG_TOKEN}}"
+
+updates:
+  - package-ecosystem: npm
+    directory: "/"
+    schedule:
+      interval: daily
+      time: "00:00"
+      timezone: UTC
+    open-pull-requests-limit: 10
+    ignore:
+    - dependency-name: husky
```

Comment:
I think this is no longer needed

Reply:
Good point, let's open a separate PR for that!